configure.ac

```diff
@@ -22,7 +22,7 @@ AC_SUBST(ODP_VERSION_API_MINOR)
 ##########################################################################
 m4_define([odph_version_generation], [1])
 m4_define([odph_version_major], [0])
-m4_define([odph_version_minor], [2])
+m4_define([odph_version_minor], [3])
```

Comment:
version change looks strange here.

Reply:
The duplicate ODPH_UNUSED macro was removed; hence the version change.